source/server-selection/server-selection.rst

```diff
@@ -1684,6 +1686,8 @@ References
 Changes
 =======
 
+2019-05-20: Added rule to ignore read preference for standalone servers
```

Comment:
"to never send read preference for standalone servers". Optionally: "standalone server types" to avoid any ambiguity about topology/server types.

Reply:
Done